client/src/MIDI.js

```diff
@@ -36,9 +36,9 @@ const handleAvailableOutputs = action('handleAvailableOutputs', (outputs) => {
   for (const output of outputs) {
     if (store.selectedOutputKey === output.key) return
   }
-  if (outputs.length) {
+  if (outputs.length && !store.selectedOutputKey) {
     store.selectedOutputKey = outputs[0].key
-  } else {
+  } else if (!store.length) {
```

Comment:
This should be `outputs.length`?

Reply:
Yep, that was a typo.  I'm gonna revert this change as it's unnecessary anyway.